scripts/get

```diff
@@ -83,26 +83,77 @@ latest_version() {
     fi
 }
 
-download_and_install() {
-    GCB_URL=https://storage.googleapis.com/k8s-conform-cri-o/artifacts
-    TARBALL_URL=$GCB_URL/cri-o.$ARCH.$VERSION.tar.gz
-
-    TMPDIR="$(mktemp -d)"
-    trap 'rm -rf -- "$TMPDIR"' EXIT
-
-    echo "Downloading $TARBALL_URL to $TMPDIR"
-    curl -sSfL "$TARBALL_URL" | tar xfz - --strip-components=1 -C "$TMPDIR"
-
-    echo Installing CRI-O
-    make -C "$TMPDIR"
-
-}
-
-main() {
+prepare() {
     parse_args "$@"
     verify_requirements
     latest_version
-    download_and_install
 }
 
-main "$@"
+prepare "$@"
+
+GCB_URL=https://storage.googleapis.com/k8s-conform-cri-o/artifacts
+TARBALL_URL=$GCB_URL/cri-o.$ARCH.$VERSION.tar.gz
+
+TMPDIR="$(mktemp -d)"
+trap 'rm -rf -- "$TMPDIR"' EXIT
+
+echo "Downloading $TARBALL_URL to $TMPDIR"
+curl -sSfL "$TARBALL_URL" | tar xfz - --strip-components=1 -C "$TMPDIR"
+
+echo Installing CRI-O
+pushd "$TMPDIR"
+
+# In this section we include the contents of
+# contrib/bundle/install-paths and contrib/bundle/install
+
+# INCLUDE
+DESTDIR=${DESTDIR:-}
+PREFIX=${PREFIX:-/usr/local}
+ETCDIR=${ETCDIR:-/etc}
+CONTAINERS_DIR=${CONTAINERS_DIR:-$ETCDIR/containers}
+CNIDIR=${CNIDIR:-$ETCDIR/cni/net.d}
```

Comment:
are we ablet o source the contrib scripts here?

Reply:
We pipe the script through curl, which means we would have to `curl` again at this point. Do we wanna do that or stick to the current `make get-scripts` approach?

Reply:
ach, damn, of course. No that's fine. I'd like some way that we future proof these fields, so they're not inconsitent between the (now two) places they exist (top-level makefile and here). not blocking on this PR, but probably some auto generation and hack/tree_status.sh will be the method...

Reply:
We have it via `make get-script`, the new GitHub actions job will verify that the content is sane:
https://github.com/cri-o/cri-o/pull/4610/checks?check_run_id=2022361523

:)

Reply:
then the only thing that needs fixing is my ability to read :)